packages/dd-trace/src/config.js

```diff
@@ -16,7 +16,7 @@ const telemetryMetrics = require('./telemetry/metrics')
 const { isInServerlessEnvironment, getIsGCPFunction, getIsAzureFunction } = require('./serverless')
 const { ORIGIN_KEY } = require('./constants')
 const { appendRules } = require('./payload-tagging/config')
-const { getEnvironmentVariable, getEnvironmentVariables } = require('./config-helper')
+const { getEnvironmentVariable: getEnv, getEnvironmentVariables } = require('./config-helper')
```

Comment:
I see that you do this in a few other files as well, which I understand. But why don't you just rename it in `config-helper.js` if you think it's too long of a name?